Vic2ToHoI4/Source/HOI4World/HoI4Country.cpp

```diff
@@ -531,26 +532,22 @@ void HoI4::Country::setGovernmentToExistingIdeology(const std::set<std::string>&
 }
 
 
-void HoI4::Country::addLeader(Names& names, graphicsMapper& theGraphics)
+void HoI4::Country::createLeader(Names* names, graphicsMapper* theGraphics)
```

Comment:
Wouldn't smart pointers be better?

Reply:
I was applying guidance from my employer about how best to make it clear at the call site that a function is modifying a parameter. Your comment led me to look up the guidance, only to find it's been deprecated. So I've gone back to non-const references